### PR TITLE
Packages: Extract `@wordpress/nux` package

### PR DESCRIPTION
**Pull request by gziolo**
## Description
Part of #3955.

This PR extracts new `@wordpress/nux` package.

## How has this been tested?

`npm test` and make sure there are no regressions.

Make sure you can see the tips when you load new post (make sure you have `localstorage` cleared before).

### CODE REVIEW

File: packages/nux/README.md [hunk 1]
```diff
@@ -1,12 +1,19 @@
-NUX (New User eXperience)
-=========================
+# @wordpress/nux
 
-The NUX module exposes components, and `wp.data` methods useful for onboarding a new user to the WordPress admin interface. Specifically, it exposes _tips_ and _guides_.
+The NUX (New User eXperience) module exposes components, and `wp.data` methods useful for onboarding a new user to the WordPress admin interface. Specifically, it exposes _tips_ and _guides_.
```
Comment (tofumatt) [inline packages/nux/README.md]: ❤️

**Review by tofumatt — CHANGES_REQUESTED** [review]
You should add an eslint rule to the `.eslintrc` that forbids imports from `'nux'` and `'nux/*'`, similar to the other package import rules.

**Comment by gziolo** [comment]
@tofumatt,  an eslint rule added with f6829aa.

**Comment by tofumatt** [comment]
> Edit: There was a similar rule already, I updated to use the more common pattern.

Oops, I didn't realise!

**Comment by gziolo** [comment]
Thre is still some issue with loading CSS file with this config. I'm looking into it.

**Comment by gziolo** [comment]
663f9d1 fixes the issue with missing CSS files. It's not great but should help us move forward with this one. It's exactly the same approach we took when moving `components` folder in #7640:
https://github.com/WordPress/gutenberg/blob/330a4cf23b3fba073038e4bab041ff612bef09c4/components/index.js

Let's open a follow up PR where we tackle copying CSS files generated by packages build step and add RTL support there, too.

**Review by youknowriad — APPROVED** [review]
LGTM 👍

**Review by tofumatt — APPROVED** [review]

**Comment by gziolo** [comment]
Opened #8067 to keep track of CSS workaround.

**Pull request merged**